### DISCUSSION TITLE
Account creation?

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [FlagClicked/FlagClicked#75] [Help] [answered]
Is there a current way to make an account? Or is the log in there for no reason?

**Comment by FunctionalMetatable** [comment, marked as answer]
http://www.whenflagclicked.org/login

**Reply by Steve0Greatness** [reply]
Is it supposed to still say login when I finished?

**Reply by FunctionalMetatable** [reply]
> Is it supposed to still say login when I finished?

yes

**Reply by Steve0Greatness** [reply]
> > Is it supposed to still say login when I finished?
> 
> yes

Is it because the account system isn't finished?

**Reply by FunctionalMetatable** [reply]
> > > Is it supposed to still say login when I finished?
> > 
> > 
> > yes
> 
> Is it because the account system isn't finished?

yeah. after full nuxtification it should work properly
